app/src/main/res/layout/activity_parking_stay.xml

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/background_parking_stay"
+    tools:context=".activity.ParkingStayActivity">
+
+    <TextView
+        android:id="@+id/start_date_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:fontFamily="sans-serif-black"
+        android:text="@string/date_start_text"
+        android:textAppearance="@style/TextAppearance.AppCompat.Body2"
+        android:textColor="#FFFEFE"
+        app:layout_constraintBottom_toTopOf="@id/end_date_text_view"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toLeftOf="@id/start_time_selected_text_view"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/start_date_selected_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:ems="5"
+        android:fontFamily="sans-serif-black"
+        android:text="@string/start_date_selected_text_view"
+        android:textAlignment="center"
+        android:textColor="#FFFEFE"
+        app:layout_constraintBottom_toTopOf="@id/start_time_selected_text_view"
+        app:layout_constraintHorizontal_bias="0.575"
```

Comment:
this property its not necessary, remove pls

Reply:
done